src/Raven.Studio/typescript/components/common/shell/setup.ts

```diff
@@ -21,4 +22,5 @@ export function initRedux() {
 
     databasesManager.default.onUpdateCallback = throttledUpdateReduxStore;
     activeDatabaseTracker.default.database.subscribe((db) => globalDispatch(activeDatabaseChanged(db?.name ?? null)));
+    clusterTopologyManager.default.localNodeTag.subscribe((nodeTag) => globalDispatch(localNodeTagChanged(nodeTag)));
```

Comment:
ok let's leave it like that but put note to fetch that from redux (we should create seprate slice for cluster info).